seatunnel-e2e/seatunnel-flink-connector-v2-e2e/src/test/resources/container-license-acceptance.txt

```diff
@@ -0,0 +1 @@
+mcr.microsoft.com/mssql/server:2022-latest
```

Comment:
Why do you need to add this file in this location

Reply:
the sqlserver container in testcontainers need this file